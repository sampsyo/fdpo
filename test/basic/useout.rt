in x: 32;
out y: 32;
out z: 32;
y = add[32](x, 32d1);
z = add[32](y, 32d1);
