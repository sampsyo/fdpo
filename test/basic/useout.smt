(let ((.def_0 (bvadd y #b00000000000000000000000000000001))) (let ((.def_1 (= z .def_0))) (let ((.def_2 (bvadd x #b00000000000000000000000000000001))) (let ((.def_3 (= y .def_2))) (let ((.def_4 (and .def_3 .def_1))) .def_4)))))
