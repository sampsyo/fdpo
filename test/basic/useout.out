y = 101
z = 102
